app/assets/stylesheets/layout/navbar.scss

```diff
@@ -177,6 +181,12 @@ header#main_header {
     }
   }
 
+  .navbar-inverse, .navbar-toggle:hover {
+    margin-left:5px;
+    margin-top:5px;
```

Comment:
Making this margin-top be margin-top:8px; makes the burger jumping problem go away.